packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -235,6 +235,16 @@ class TextEditingController extends ValueNotifier<TextEditingValue> {
 /// is a full-featured, material-design text input field with placeholder text,
 /// labels, and [Form] integration.
 ///
+/// ## Gesture Events Handling
+///
+/// This widget will provide rudimentary, platform-agnostic gesture event
+/// handling for user actions such as tapping, long pressing and scrolling when
+/// [rendererIgnoresPointer] is false (false by default). To tightly conform
+/// to the platform behavior with respect to gesture events in text fields, use
+/// [TextField] or [CupertinoTextField]. For custom selection behavior, call
+/// APIs such as [RenderEditable.selectPosition], [RenderEditable.selectWord]
+/// etc programmatically.
```

Comment:
etc => etc.,

Reply:
good point, I never thought about this. Thanks